### DISCUSSION TITLE
can anyone explain how he outfit editor works

### DISCUSSION THREAD

**Discussion by snwyyy** [YimMenu/YimMenu#1541] [Q&A]
idk how it works and i see like XDEV outfit editor and im trying to replicate outfits from that but idk what the undershirt or something like that is for this menu

**Comment by chenry0553** [comment]
You have to play with a little bit but this site helped me find the outfit ID's for each slot I was looking for. I made a police officer outfit, and several other funny outfits once I found the ID's. 

https://polarized0.github.io/gta5_outfit_builder/

**Reply by snwyyy** [reply]
can u tell me the IDS for the police outfit

**Reply by snwyyy** [reply]
and also how can i wear undershirt and normal shirt at the same time?

**Comment by LintStalker** [comment]
Is it possible to swipe the out fit of a nearby character?

**Comment by LintStalker** [comment]
Something cool that you can do with this is  saving outfits that you get while doing a mission.  I was doing the ULP missions, and I saved one of the IAA outfits.
